frontend/src/app/components/modals/create-namespace-bucket-modal/create-namespace-bucket-modal.js

```diff
@@ -118,6 +119,8 @@ class CreateNamespaceBucketModalViewModel extends ConnectableViewModel {
         const readPolicy = getFieldValue(form, 'readPolicy');
         const writePolicy = getFieldValue(form, 'writePolicy');
         const cacheTTL = getFieldValue(form, 'cacheTTL');
+        const cacheEvictionType = getFieldValue(form, 'cacheEvictionType');
```

Comment:
A better way to read more than one or two values from a form is to use `getFormValues` to fetch all the values at once instead of fetching them one by one.

```suggestion
        const { readPolicy, writePolicy, cacheTTL, cacheEvictionType } = getFormValues(form);
```

Reply:
Removed my changes